packages/shared/src/utils/constants/error.ts

```diff
@@ -5,3 +5,12 @@ export const getDefaultError = () => ({
     description: localize('Our servers hit a bump. Let’s refresh to move on.'),
     cta_label: localize('Refresh'),
 });
+
+export const STATUS_CODES = Object.freeze({
```

Comment:
`as const` is preferred over freeze
ref: https://youtu.be/jjMbPt_H3RQ

Reply:
Yes they're not enums, yet if we know we'll not modify this object in any case within our code, isn't it unnecessary to freeze the object, 
it's your choice